src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -1899,7 +1969,7 @@ export default class AircraftModel {
     _calculateTargetedAltitudeVnavClimb(waypointWithMinimumAltitudeRestriction) {
         const waypointMinimumAltitude = waypointWithMinimumAltitudeRestriction.altitudeMinimum;
```

Comment:
if we're just returning a re-assigned property here, we can probably remove this method entirely and just use `waypointWithMinimumAltitudeRestriction.altitudeMinimum` in it's place.